content/en/style-guide/_index.md

```diff
@@ -151,40 +151,40 @@ Note that we cannot link to content developed by vendors.
 
 #### Keeping it simple
 
-The Glossary aims at **explaining complex concepts in simple words** — that is a surprisingly difficult task that will likely take multiple revisions. 
+The Glossary aims to **explain complex concepts in simple words**—a surprisingly difficult task that will likely take multiple revisions. 
 Always keep the audience in mind when drafting your definition. 
-Avoid using industry terms and buzzwords — you'll probably catch yourself going back to them and may need to autocorrect. 
```

Comment:
I'm not sure that needing to autocorrect is the correct phrasing we want here. I know it's a bit of a whimsical way of telling folks to check that they aren't in a bit of a rut, but autocorrect is a specific thing. Happy to hear thoughts on this.

Reply:
Hi @nate-double-u 
I agree with you. "need to autocorrect" is not an intuitive description to me.

Reply:
Sure. How would you phrase it?

Reply:
I've pushed an update—let me know what you think.